docs/Getting-Started.md

```diff
@@ -24,30 +29,13 @@ fastify.listen(3000, function (err) {
 })
 ```
 
-Do you prefer to use `async/await`? Please review [Routes#async-await](https://github.com/fastify/fastify/blob/master/docs/Routes.md#async-await)!
-
-<a name="schema"></a>
-### Schema serialization
-Fastify is designed for performance. To truly turbocharge our server, we can serialize responses according to the [JSON Schema](http://json-schema.org/) standard:
+Do you prefer to use `async/await`? We got you covered.  
+*(we also suggest to use [make-promises-safe](https://github.com/mcollina/make-promises-safe) to avoid file descriptor and memory leaks)*
 ```js
 const fastify = require('fastify')()
 
-const opts = {
-  schema: {
-    response: {
-      200: {
-        type: 'object',
-        properties: {
-          hello: { type: 'string' }
-        }
-      }
-    }
-  }
-}
-
-// Declare a route with an output schema
-fastify.get('/', opts, function (request, reply) {
-  reply.send({ hello: 'world' })
+fastify.get('/', async (request, reply) => {
```

Comment:
Does the `reply` parameter even need to be supplied?

Reply:
Yes, if you need to set some custom header o change the status code.

Reply:
Then maybe a note about it and why it isn't being used here would be good to have.

Reply:
I agree that a note could be a nice idea, but I think that this document should provide just a getting started, we have provided the link to other parts of the documentation that explains better the usage of every api. I don't think that add here too many info could be a good idea.